scripts/export-pbrify-onnx.py

```
@@ -0,0 +1,93 @@
+MODELS = {
+    "normal":    "1x-PBRify_NormalV3",
+    "roughness": "1x-PBRify_RoughnessV2",
+    "height":    "1x-PBRify_Height",
+}
+BASE_URL = "https://github.com/Kim2091/PBRify_Remix/raw/main/Models/{name}.pth"
+
+
+def download(name: str, dest: str) -> None:
+    url = BASE_URL.format(name=name)
+    print(f"  downloading {url}")
+    urllib.request.urlretrieve(url, dest)
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)